src/stream/src/executor/hash_join.rs

```diff
@@ -878,10 +879,11 @@ mod tests {
         // push the 1st left chunk
         tx_l.push_chunk(chunk_l1);
         let chunk = hash_join.next().await.unwrap().unwrap();
-        assert_eq!(
-            chunk.into_chunk().unwrap(),
-            StreamChunk::from_pretty("I I I I")
-        );
+
+        assert!(chunk
```

Comment:
Todo: make this have a human readable error message (maybe change to `assert_semantically_equivalent_to`